src/app/_components/timeline.tsx

```diff
@@ -0,0 +1,85 @@
+// components/Timeline.tsx
+import React from "react";
+import Image from "next/image";
+import tm1 from "public/images/tm1.jpg";
+import tm2 from "public/images/tm2.jpg";
+import tm3 from "public/images/tm3.jpg";
+import tm4 from "public/images/tm4.jpg";
+
+const Timeline: React.FC = () => {
+  const eventSections = [
+    { title: "Registration", time: "9:00 AM" },
```

Comment:
Move events to external file (e.g. json) to prevent accidental editing of the page when updating the info.

Reply:
done!